lib/components/icons/Icon/Icon.migration.md

```diff
@@ -0,0 +1,13 @@
+# Icon Migration Guide
+
+## API Changes
+
+- If you need a custom icon size, use `size="fill"` to make the icon fill its container.
```

Comment:
Worth mentioning how `inline`works? It solves a bunch of problems that we used to have.